src/main/scala/com/azavea/modellab/Service.scala

```diff
@@ -93,12 +135,13 @@ object Service extends SimpleRoutingApp with DataHubCatalog with Instrumented wi
             for {
               name <- breaksName
               breaks <- colorBreaks.get(name)
-            } yield tile.renderPng(breaks).bytes
+              // noDataColors.getOrElse outside comprehension b/c lacking `map` implementation
+            } yield tile.renderPng(breaks, noDataColors.getOrElse(name, 0)).bytes
```

Comment:
I don't think this appropriate. In NDVI case (-1 to 1) 0 should not be default NoData. Probably better to do something like:
```scala
} yield noDataColors.get(name) match {
  case Some(noDataColor) => tile.renderPng(breaks, noDataColor).bytes
  case None => tile.renderPng(breaks).bytes
```
All this nested matching sure is annoying. Wish there was a cleaner way to translate it into code.